contracts/utils/deploy.js

```diff
@@ -170,6 +328,11 @@ const _verifyProxyInitializedWithCorrectGovernor = (transactionData) => {
     return;
   }
 
+  if (isMainnet || isBase || isFork) {
+    // TODO: Skip verification for Fork for now
+    return;
+  }
+
```

Comment:
We actually don't need this, I was doing this to test the crosschain strategy on fork